hackathon_site/event/test_api.py

```diff
@@ -1,19 +1,21 @@
+from datetime import datetime
+
 from django.contrib.auth.models import Group
 from django.urls import reverse
+from django.conf import settings
 from rest_framework import status
 from rest_framework.test import APITestCase
 from hackathon_site.tests import SetupUserMixin
 from django.contrib.auth.models import Permission
 
-
 from event.models import Profile, User, Team
 from event.serializers import (
     UserSerializer,
     TeamSerializer,
 )
 
-from hardware.serializers import OrderListSerializer
-from hardware.models import Hardware, Order, OrderItem
+from hardware.serializers import OrderListSerializer, IncidentSerializer
```

Comment:
unused imports here, but this file should not have changes, you can revert this file to match what's on develop by running
`git checkout develop -- hackathon_site/event/test_api.py` in the hackathon-template directory